webcurator-core/src/main/java/org/webcurator/core/store/DigitalAssetStoreClient.java

```diff
@@ -161,11 +161,7 @@ public HarvestResultDTO copyAndPrune(String targetInstanceName, int originalHarv
                                          List<String> urisToDelete, List<HarvestResourceDTO> harvestResourcesToImport) throws DigitalAssetStoreException {
         HarvestStoreCopyAndPruneDTO dto = new HarvestStoreCopyAndPruneDTO();
         dto.setUrisToDelete(urisToDelete);
-        if (harvestResourcesToImport != null) {
-            dto.setHarvestResourcesToImport(harvestResourcesToImport.stream().map(o -> {
-                return (ArcHarvestResourceDTO) o;
-            }).collect(Collectors.toList()));
-        }
```

Comment:
@leefrank9527 are you certain that `harvestResourcesToImport `cannot be null?

Reply:
The null validation here is just for castting HarvestResourceDTO to ArcHarvestResourceDTO. If harvestResourcesToImport is null, it will not be fed to dto, the harvestResourcesToImport in dto is still null. Currently the input harvestResourcesToImport from Webapp and AutoProune Task aren't null. It's safe to remove the  null validation logics here.